File: doc/src/scripts/ControlFromHere.cpp
#include <iostream>
#include <krpc.hpp>
#include <krpc/services/space_center.hpp>

int main() {
  krpc::Client conn = krpc::connect();
  krpc::services::SpaceCenter space_center(&conn);
  auto vessel = space_center.active_vessel();
  auto part = vessel.parts().with_title("Clamp-O-Tron Docking Port").front();
  vessel.parts().set_controlling(part);
}

File: doc/src/scripts/ControlFromHere.cs
using System;
using System.Linq;
using KRPC.Client;
using KRPC.Client.Services.SpaceCenter;

class ControlFromHere
{
    public static void Main ()
    {
        var conn = new Connection ();
        var vessel = conn.SpaceCenter ().ActiveVessel;
        var part = vessel.Parts.WithTitle ("Clamp-O-Tron Docking Port") [0];
        vessel.Parts.Controlling = part;
    }
}
